launchable/commands/record/build.py

```diff
@@ -183,6 +191,17 @@ def build(ctx: click.core.Context, build_name: str, source: List[str], max_days:
     uniq_submodules = {commit_hash: (name, repo_dist, commit_hash)
                        for name, repo_dist, commit_hash, in sources + submodules}.values()
 
+    if no_commit_collection and len(branches) != 0:
+        branch_name_map = dict(normalize_key_value_types(branches))
```

Comment:
[must]
If you want to use `normalize_key_value_types` here, please fix the comment in the method that is saying it is for flavor.
https://github.com/launchableinc/cli/blob/4344bd0463fdc4fc5a12bfa3e1d86232be2f4d9c/launchable/utils/key_value_type.py#L4